tasks/build/package_scripts/pre_remove.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+echo -n "Stopping kibana service..."
+if command -v systemctl >/dev/null; then
```

Comment:
Should this also use the `&& systemctl is-active kibana.service >/dev/null` condition like you have in the `pre_install.sh` script?